shared/src/test/scala/pl/metastack/metatime/TimeTest.scala

```diff
@@ -14,18 +14,33 @@ object TimeTest extends SimpleTestSuite with TestIgnored {
   }
 
   testIgnored("hour") {
-    assertEquals(Hour(25).days, Day(1 + 1.0 / 24))
+    assertEquals(Hour(25).days, Day(1 + 1 / 24))
```

Comment:
We could also change `Day` to a double value, so that we can format the fractional part like this:

```scala
val d = Day(1 + 1.0 / 24)
s"${d.value} days and ${d.hours.value} hours"
```

One problem with this approach is that in order to get the minutes, the user needs to call `d.hours.minutes.value` (`d.minutes.value` would be 60). What do you think?